templates/blm-ce.html

```diff
@@ -65,253 +63,311 @@
+              {{categoricalExclusionJustification}}
+            </p>
+          </li>
+
+          <li>
+            <h3>Signature</h3>
+            <div aria-label="Signature area">>>> signature goes here</div>
+          </li>
+        </ol>
+      </main>
+      <footer role="contentinfo">
+        <section class="mt-3">
           <p>
-            This categorical exclusion is appropriate in this situation because there are no extraordinary circumstances
-            having effects that significantly affect the environment. The proposed action has been reviewed, and none of the
-            extraordinary circumstances described in 43 CFR Part 46.215 apply.
-            I considered {{categoricalExclusionJustification}}
+            <strong>Responsible Official:</strong> {{responsibleOfficial}}
+            <strong>Date:</strong> {{approvalDate}}
           </p>
-        </li>
-        
-        <li>
-          <h3>Signature</h3>
-          <div aria-label="Signature area">
-            >>> signature goes here
-          </div>
-        </li>
-      </ol>
-    </main>
-    <footer role="contentinfo">
-      <section class="mt-3">
-        <p><strong>Responsible Official:</strong> {{responsibleOfficial}} <strong>Date:</strong> {{date}}</p>
-      </section>
-      
-      <section class="mt-3">
-        <h3 class="font-bold">Contact Person</h3>
-        <p>For additional information concerning this CX review, please contact:</p>
-        <address>
-          {{contactPerson}}, {{contactTitle}}<br />
-          {{officeName}} {{mailingAddress}} {{telephoneNumber}}
-        </address>
-      </section>
-      
-      <section class="mt-3">
-        <h3 class="font-bold">ATTACHMENTS:</h3>
-        <p>List any other documents including but not limited to map(s), design features.</p>
-      </section>
-    </footer>
-  </div>
-</body>
+        </section>
+
+        <section class="mt-3">
+          <h3 class="font-bold">Contact Person</h3>
+          <p>
+            For additional information concerning this CX review, please
+            contact:
+          </p>
+          <address>
+            {{contactPerson}}, {{contactTitle}}<br />
+            {{officeName}} {{mailingAddress}} {{telephoneNumber}}
+          </address>
+        </section>
+
+        <section class="mt-3">
+          <h3 class="font-bold">ATTACHMENTS:</h3>
+          <p>
+            List any other documents including but not limited to map(s), design
+            features.
+          </p>
+          {% if attachments %}
+            <div class="attachments">
+              {% for attachment in attachments %}
+                <div class="attachment-page" style="page-break-before: always;">
+                  <h4>Attachment {{ loop.index }}</h4>
+                  <img src="{{ attachment }}"
+                      alt="Attachment {{ loop.index }}"
+                      style="max-width: 100%; height: auto;" />
+                </div>
+              {% endfor %}
```

Comment:
This assumes that attachments are images. This is a dangerous assumption to be rectified in the near future.